### PR TITLE
refactored NonFinalAttribute

### PR DESCRIPTION
**Pull request by Vitaly-Protasov**
Solved in case of #528

### CODE REVIEW

File: aibolit/patterns/non_final_attribute/non_final_attribute.py [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
 
 
 class NonFinalAttribute:
+    '''
+    return lines of non-final attributes
+    '''
+    def _check_final(self, field: ASTNode) -> bool:
+        if 'final' not in field.modifiers:
+            return True
+        return False
```
Comment (aravij) [inline aibolit/patterns/non_final_attribute/non_final_attribute.py:35]:
```suggestion
        return 'final' not in field.modifiers
```

File: aibolit/patterns/non_final_attribute/non_final_attribute.py [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
 from aibolit.utils.ast_builder import build_ast
+from typing import List
+from aibolit.ast_framework.ast_node import ASTNode
 
 
 class NonFinalAttribute:
+    '''
+    return lines of non-final attributes
+    '''
+    def _check_final(self, field: ASTNode) -> bool:
```
Comment (aravij) [inline aibolit/patterns/non_final_attribute/non_final_attribute.py:32]: If we will make this function one liner, we may just substitute its code right into `value` function.

File: aibolit/patterns/non_final_attribute/non_final_attribute.py (hunk 1 above)
Comment (Vitaly-Protasov) [reply aibolit/patterns/non_final_attribute/non_final_attribute.py]: got it

File: aibolit/patterns/non_final_attribute/non_final_attribute.py (hunk 2 above)
Comment (Vitaly-Protasov) [reply aibolit/patterns/non_final_attribute/non_final_attribute.py]: done

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**